examples/test7-compose.yaml

```diff
@@ -28,7 +28,11 @@ services:
           ip: 10.0.1.1/16
           mac: 00:0a:0b:0c:0d:01
           mtu: 4111
-          netem: "delay 40ms rate 10mbit"
+          netem: "rate 10mbit"
         - bridge: s2
           ip: 100.0.1.1/16
           dev: eth1
+
+x-network:
+  links:
+    - {service: node, bridge: s1, netem: "delay 40ms"}
```

Comment:
Do we want a comment as documentation on how this interacts with the netem setting above?

Reply:
Oh yes, certainly.